### PR TITLE
Added README files for nested projects

### PR DESCRIPTION
**Pull request by nsurgeon**
Update project folders with README.md files. Each file includes simple description and documentation references

### CODE REVIEW

File: 3d-venues-and-indoor-routing/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Prerequisites
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+Update manifest file with proper licens key as described above in [root README file](../README.md).
+=======
+Update manifest file with proper licens key as described above in root [README file](../README.md).
+>>>>>>> MOBILESDK-7817: add Read me file for venue 3d sample
+=======
+Update manifest file with proper license key as described above in root [README file](../README.md).
```
Comment (starand) [inline 3d-venues-and-indoor-routing/README.md]: please fix that

File: 3d-venues-and-indoor-routing/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## Example
+
+For testing purposes you may use Alexa Shopping Mall (Venue ID: DM_7171). Enter the value in text field and tap "OPEN" button.
+After Venue is open you can navigate inside. To build route tap "ROUTING", then tap "Start location" and pick start position, tap on "End location" and choose end position. Finally tap "SHOW ROUTE" and you will see the route on a screen.
+
+Look at code and [SDK API Documentation](https://developer.here.com/documentation/android-premium/dev_guide/topics/maps-venue-3d.html) to get more details.
+
+## References
+
+<<<<<<< HEAD
```
Comment (starand) [inline 3d-venues-and-indoor-routing/README.md]: please fix

File: 3d-venues-and-indoor-routing/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+=======
+To find more venues use [search fetures in sdk](https://developer.here.com/documentation/android-premium/dev_guide/topics/places.html) and check sample [search app](../search/README.md).
+=======
+To find more venues use [search features in sdk](https://developer.here.com/documentation/android-premium/dev_guide/topics/places.html) and check sample [search app](../search/README.md).
+>>>>>>> Splell fixes
+=======
+To find more venues use [search features in SDK](https://developer.here.com/documentation/android-premium/dev_guide/topics/places.html) and check sample [search app](../search/README.md).
+>>>>>>> Update README.md
+
+>>>>>>> Splell fixes
```
Comment (starand) [inline 3d-venues-and-indoor-routing/README.md]: fix

File: speed-limit-watcher/README.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ First, please open the `AndroidManifest.xml` file and add some permissions that
 *We will not focus on requesting permissions itself but the code example in the repository includes a requesting method that can be used for `SDK Version > 22`.  
 More information can be found [here](https://developer.here.com/documentation/android-premium/dev_guide/topics/request-android-permissions.html)*
 
-As you can also read in our [documentation](https://developer.here.com/documentation/android-premium/dev_guide/topics/app-simple.html),
+As you can also read in our [documentation](https://developer.here.com/documentation/android-premium/dev_guide/topics/app-run-simple.html),
```
Comment (starand) [inline speed-limit-watcher/README.md]: was that changed intentionally?

File: speed-limit-watcher/README.md [hunk 5]
```diff
@@ -70,7 +70,7 @@ The last snippet that we will add to the `AndroidManifest.xml` file, is the defi
 
 ## Step 2. Initializing HERE SDK 
 
-The documentation section, that we used for [creating simple app](https://developer.here.com/documentation/android-premium/dev_guide/topics/app-simple-android-studio.html) also describes how to initialize the Map Fragment.
+The documentation section, that we used for [creating simple app](https://developer.here.com/documentation/android-premium/dev_guide/topics/app-create-simple.html) also describes how to initialize the Map Fragment.
```
Comment (starand) [inline speed-limit-watcher/README.md]: same

File: speed-limit-watcher/README.md (hunk 4 above)
Comment (nsurgeon) [reply speed-limit-watcher/README.md]: no

**Pull request merged**